site/content/docs/5.2/examples/form/index.html

```diff
@@ -0,0 +1,384 @@
+                  company</label>
+                <input type="text" class="form-control" id="inputCompany"
+                  aria-labelledby="inputCompanyLabel inputCompanyFeedback" required>
+                <div class="valid-feedback">
+                  Looks good!
+                </div>
+                <div id="inputCompanyFeedback" class="invalid-feedback">
+                  Please enter a valid company name.
+                </div>
+              </div>
+              <div class="mb-3">
+                <label for="selectBusiness" id="selectBusinessLabel" class="form-label is-required">Business
+                  type</label>
+                <select class="form-select" id="selectBusiness"
+                  aria-labelledby="selectBusinessLabel selectBusinessFeedback" required>
+                  <option selected disabled value="" aria-hidden="true">Select a type</option>
+                  <option value="1">Sole proprietorship</option>
+                  <option value="2">Partnership</option>
+                  <option value="3">Corporation</option>
+                  <option value="4">Limited liability company</option>
+                </select>
+                <div class="valid-feedback">
+                  Looks good!
+                </div>
+                <div id="selectBusinessFeedback" class="invalid-feedback">
+                  Please select a valid business type.
+                </div>
+              </div>
+              <div class="mb-3">
+                <label for="textareaMessage" id="textareaMessageLabel" class="form-label is-required">Message</label>
+                <button type="button" class="btn btn-link p-0 mb-1" data-bs-toggle="tooltip" data-bs-placement="top"
+                  data-bs-title="Help message for textarea">
+                  <svg xmlns="http://www.w3.org/2000/svg" width="1.25rem" height="1.25rem" focusable="false"
+                    aria-hidden="true">
+                    <use xlink:href="assets/img/boosted-sprite.svg#tooltip"></use>
+                  </svg>
+                  <span class="visually-hidden">Help message for textarea</span>
+                </button>
+                <textarea class="form-control" id="textareaMessage" rows="5"
+                  placeholder="Describe your business and your motivation in becoming on Orange reseller"
+                  aria-labelledby="textareaMessageLabel textareaMessageFeedback" required></textarea>
+                <div class="valid-feedback">
+                  Looks good!
+                </div>
+                <div id="textareaMessageFeedback" class="invalid-feedback">
+                  Please select a valid business type.
+                </div>
+              </div>
+              <button type="submit" class="btn btn-primary mt-2">Submit</button>
+            </div>
+          </div>
+        </form>
+      </div>
+    </div>
+  </div>
+</main>
+<!-- Footer -->
+<footer class="footer bg-dark navbar-dark">
```

Comment:
The footer should be the same as [in the DSM](https://system.design.orange.com/0c1af118d/p/20500e-form/b/16bb53) (except the year)

![Screenshot 2022-11-10 at 08 36 42](https://user-images.githubusercontent.com/17381666/201028558-8410b7e7-2efa-4509-b762-2f6ec33b14d4.png)